torchvision/models/detection/keypoint_rcnn.py

```diff
@@ -55,10 +57,10 @@ class KeypointRCNN(FasterRCNN):
             If box_predictor is specified, num_classes should be None.
         min_size (int): minimum size of the image to be rescaled before feeding it to the backbone
         max_size (int): maximum size of the image to be rescaled before feeding it to the backbone
-        image_mean (Tuple[float, float, float]): mean values used for input normalization.
+        image_mean (List[float, float, float]): mean values used for input normalization.
```

Comment:
Well we have decided it to be `List[int]` let me know if this is fine.

Reply:
You mean `List[float]`, right? Plus, `List` only has a single inner type.

```suggestion
        image_mean (List[float]): mean values used for input normalization.
```

Reply:
See my comment below for using Tuple.